internal/levee_test.go

```diff
@@ -29,10 +30,18 @@ var patterns = []string{
 	"example.com/tests/sinks",
 }
 
+var debugging bool
```

Comment:
I agree. I've added a flag. Is this the correct way to do it?

Reply:
This is fine.  Since this is self-contained, there's no need to defer evaluation to `init()`, and you could've done
```golang
var debugging *bool = flag.Bool("debug", false, "run the debug analyzer")
```

It boils down to style and whether or not you want to deal with a `*bool`.